web/modules/custom/demo_content/demo_content.module

```diff
@@ -11,17 +11,24 @@ use Drupal\og\OgMembershipInterface;
 
 /**
  * Implements hook_ENTITY_TYPE_create().
+ *
+ * We have two organic groups, the collection and the solution, and two users
+ * intended to be used as facilitators, the collection_facilitator and the
+ * solution_facilitator respectively.
+ *
+ * This function will assign the appropriate user as a facilitator to the entity
+ * passed.
```

Comment:
Is it good though to use 'We'? because there are really a lot of cases that I find it really hard to come up with a way to describe it adjectively and documenting with what 'we' do is so much easier to explain. I mean in terms of best practices.